v1/sensor3700.js

```diff
@@ -152,4 +152,14 @@ module.exports = function (RED) {
     });
   }
   RED.nodes.registerType('sensor3700 in', SensorNode);
+  SensorNode.prototype.close = function () {
```

Comment:
Geriau jei `registerType` būtų vykdoma po to kai SensorNode klasės prototipas būna galutinai užbaigtas. Dabar, nors ir be galo maža, egzistuoja race-condition  tikimybė.